Makefile

```diff
@@ -26,23 +26,42 @@ staticcheck:
 exhaustive:
 	@sh -c "'$(CURDIR)/scripts/exhaustive.sh'"
 
-website:
-	@echo "==> Downloading latest Docker image..."
-	@docker pull hashicorp/terraform-website:full
-	@echo "==> Starting core website in Docker..."
-	@docker run \
-		--interactive \
+PWD=$$(pwd)
+DOCKER_IMAGE="hashicorp/terraform-website:full"
+DOCKER_IMAGE_LOCAL="terraform-website-local"
+DOCKER_RUN_FLAGS=--interactive \
 		--rm \
 		--tty \
 		--workdir "/website" \
 		--volume "$(shell pwd):/website/ext/terraform" \
+		--volume "$(shell pwd)/website:/website/preview" \
 		--publish "3000:3000" \
-		hashicorp/terraform-website:full \
-		npm start
+		-e "IS_CONTENT_PREVIEW=true" \
+		-e "PREVIEW_FROM_REPO=terraform" \
+		-e "NAV_DATA_DIRNAME=./preview/data" \
+		-e "CONTENT_DIRNAME=./preview/docs" \
+		-e "CURRENT_GIT_BRANCH=$$(git rev-parse --abbrev-ref HEAD)"
```

Comment:
style: It might make sense to put these at the top with the other variable declarations?